pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -219,13 +219,7 @@ type ScorePlugin interface {
 	// indicating the rank of the node. All scoring plugins must return success or
 	// the pod will be rejected.
 	Score(pc *PluginContext, p *v1.Pod, nodeName string) (int, *Status)
-}
 
-// ScoreWithNormalizePlugin is an interface that must be implemented by "score"
-// plugins that also need to normalize the node scoring results produced by the same
-// plugin's "Score" method.
-type ScoreWithNormalizePlugin interface {
-	ScorePlugin
 	// NormalizeScore is called for all node scores produced by the same plugin's "Score"
 	// method. A successful run of NormalizeScore will update the scores list and return
 	// a success status.
```

Comment:
Document that implementations can return `nil` and what the semantics of it are.

Reply:
We could document as below, what do you think?

```go
This function is optional, and you could implement it as a noop by simply return nil
```

Reply:
It takes me some seconds to figure out what "noop" is... And then I realized it's "no-op", so prefer to reword to "no-op".

Reply:
> It takes me some seconds to figure out what "noop" is... And then I realized it's "no-op", so prefer to reword to "no-op".

Sorry for the confusion, I'll make the update.

Reply:
done